cnxpublishing/views/admin.py

```diff
@@ -271,3 +534,150 @@ def admin_edit_site_message_POST(request):
     args = admin_edit_site_message(request)
     args['response'] = "Message successfully Updated"
     return args
+
```

Comment:
Do you mean the function above it? The function above gets the info for the given site message and populates the form, then this function actually updates the site message with any changes made